app/uk/gov/hmrc/taxenrolmentassignmentfrontend/controllers/auth/AuthAction.scala

```diff
@@ -94,14 +97,25 @@ class AuthAction @Inject()(
           ),
           er
         )
-        Unauthorized("NoActiveSession")
+        toGGLogin
       case er: AuthorisationException =>
         logger.logEvent(
           logAuthenticationFailure(
             s"Auth exception: ${er.getMessage} for  uri ${request.uri}"
           )
         )
+        //TODO Redirect to the error page
         Unauthorized(er.getMessage)
     }
   }
+
+  def toGGLogin: Result = {
+    Redirect(
+      appConfig.loginURL,
+      Map(
+        "continue_url" -> Seq(appConfig.loginCallback),
+        "origin"   -> Seq("tax-enrolment-assignment-frontend")
+      )
+    )
+  }
```

Comment:
Is this the correct approach? Like was said this morning in refinement we aren't actually a service so a sign in shouldn't come directly back to us (as we wouldn't know where to send them back).